lib/auth/webauthn/session.go

```diff
@@ -25,7 +25,7 @@ import (
 	wan "github.com/go-webauthn/webauthn/webauthn"
 	"github.com/gravitational/trace"
 
-	wanpb "github.com/gravitational/teleport/api/types/webauthn"
+	"github.com/gravitational/teleport/api/client/proto"
```

Comment:
Maybe:

```suggestion
	authpb "github.com/gravitational/teleport/api/client/proto"
```

`proto` is a terrible package name, plus it confuses itself with the official proto imports.